### DISCUSSION TITLE
Bug when using dark mode

### DISCUSSION THREAD

**Discussion by ecoy1** [awcodes/filament-table-repeater#133] [Q&A] [answered]
Hi,

I use the Table Repeater in a form and the headers dont ever change to dark theme for some reason.  Any idea why this might be happening?

<img width="1297" alt="image" src="https://github.com/user-attachments/assets/53cb2f43-1da3-4e31-9360-cab669ab24ec">

**Comment by awcodes** [comment, marked as answer]
You need to follow the installation instructions in the readme. It appears that you didn't now setup the plugin with a custom theme for filament.

**Reply by ecoy1** [reply]
I found out that I had written `'/vendor/awcodes/filament-table-repeater/resources/**/*.blade.php'` in the tailwind.congif.js file instead of `'./vendor/awcodes/filament-table-repeater/resources/**/*.blade.php'` Once I changed it, it was working all fine. Thank you!
